src/services/ip-address/ipAddress.service.ts

```diff
@@ -0,0 +1,15 @@
+import 'reflect-metadata'
+
+import { Request } from 'express'
+import { provide } from 'inversify-binding-decorators'
+
+@provide(IpAddressService)
+export default class IpAddressService {
+
+  public getIpAddress(req: Request): string {
+    const forwardedIpsStr: string = req.headers['x-forwarded-for'] as string
+    const ip: string = forwardedIpsStr?.split(',')[0] || ''
```

Comment:
no need for the `?` or `|| ''` if we can guarantee this header will always be there.